uiautomationutilities/build.gradle

```diff
@@ -181,15 +181,19 @@ dependencies {
     implementation "androidx.test.uiautomator:uiautomator:$rootProject.ext.uiAutomatorVersion"
     implementation "androidx.test:monitor:$rootProject.ext.androidxTestMonitorVersion"
     implementation "androidx.test:core:$rootProject.ext.androidxTestCoreVersion"
-    if (findProject(":testutils") != null) {
-        //Use project reference if available
-        implementation(project(":testutils")) {
-            transitive = true
-        }
-    } else {
-        //Always use the latest release
-        implementation(group: 'com.microsoft.identity', name: 'testutils', version: '0.0.+')
+    implementation(project(":LabApiUtilities")) {
+        exclude module: 'common'
```

Comment:
I think we can remove this exclude now since LabApiUtilitities doesn't actually take a dependency on common